src/Apps/Collect/Components/ArtworkGrid/index.tsx

```diff
@@ -72,30 +90,20 @@ class Filter extends Component<Props> {
   renderWaysToBuy(filters, mediator) {
     return (
       <React.Fragment>
-        <Sans size="2" weight="medium" color="black100" mt={0.3}>
+        <Sans size="2" weight="medium" color="black100" my={1}>
           Ways to Buy
         </Sans>
-        <Checkbox
```

Comment:
Any reason to remove the 'Buy Now' checkbox?

Reply:
Yes, the idea is to remove 'Buy Now' until we actually launch since the collect page is going out to users beforehand

Reply:
Sweet. You can use the lab feature if needed to still include it for testing purposes for admins, etc.